scripts/format_elevator_data.exs

```diff
@@ -24,8 +24,9 @@ path
    %{
      "elevator_id" => id,
      "alternate_elevator_ids" => alternate,
-     "Elevator Category (Disregarding Entering/Exiting)" => category,
-     "Short Text" => summary
+     "Elevator Category" => category,
+     "Short Text" => exiting_summary,
+     "Edited Alternative Text" => summary
```

Comment:
I'm a little confused about the `exiting summary` naming vs. `summary`. If I'm understanding correctly, the Edited Alternative Text from the spreadsheet is what's more describing the exiting information. Should this instead be set to something like the following, since we will always have the Short Text, but not always the Edited Alternative Text:
 ```   
     "Short Text" => summary,
     "Edited Alternative Text" => redundancy_summary/backup_summary
```

Reply:
Given that it's still being used in the Elevator screens code, I think it's fine to leave as is